packages/graphql-api/src/graphql/mutations.ts

```diff
@@ -1,11 +1,16 @@
 export const mutations = `
-  loginWithPassword(user: String, userFields: UserInput, password: String!): LoginReturn
+  impersonate(accessToken: String! username: String!): ImpersonateReturn
   refreshTokens(accessToken: String!, refreshToken: String!): LoginReturn
   logout(accessToken: String!): Boolean
-  impersonate(accessToken: String! username: String!): ImpersonateReturn
-  createUser(user: CreateUserInput!): Boolean
+  authenticate(serviceName: String!, params: AuthenticateParamsInput!): LoginReturn
```

Comment:
I think it will be something like this (tested), not so obvious though, what do you think?:

```
mutation {
  authenticate(serviceName: "password", params: {password: "<pw>", user: {email: "<email>"}}) {
    sessionId
    tokens {
      refreshToken
      accessToken
    }
  }
}
```